packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/BarPlotMarkerConfigurationMenu.tsx

```diff
@@ -80,18 +167,93 @@ export function BarPlotMarkerConfigurationMenu({
         toggleStarredVariable={toggleStarredVariable}
         constraints={constraints}
       />
-      <RadioButtonGroup
-        containerStyles={{
-          marginTop: 20,
-        }}
-        label="Y-axis"
-        selectedOption={configuration.selectedPlotMode || 'count'}
-        options={['count', 'proportion']}
-        optionLabels={['Count', 'Proportion']}
-        buttonColor={'primary'}
-        margins={['0em', '0', '0', '1em']}
-        onOptionSelected={handlePlotModeSelection}
-      />
+      <div style={{ margin: '5px 0 0 0' }}>
+        <span style={{ fontWeight: 'bold' }}>
+          Summary marker (all filtered data)
+        </span>
+        {overlayConfiguration?.overlayType === 'categorical' ? (
+          <MarkerPreview data={overlayConfiguration} mapType="barplot" />
+        ) : (
+          continuousMarkerPreview
+        )}
+      </div>
+      <LabelledGroup label="Marker X-axis controls">
```

Comment:
The controls get very wide - on my screen the menu takes up more than half so I can't see the map's spinner any more.
I'm no CSS expert but adding this
`containerStyles={{ display: 'flex' }}`
to the `LabelledGroup`s (and the next one) at least makes them wrap down the screen. I'd recommend that as a temporary measure before addressing the styling of the controls, headings etc.

PS. The LabelledGroup component is all my fault - not sure how we would style the labels within them (e.g. "Binning method")